protoconfloader.go

```diff
@@ -0,0 +1,327 @@
+package protoconfLoader
```

Comment:
Should we change it to `protoconf_loader`?
I think package names are snake_case by convention